kong/clustering/rpc/manager.lua

```diff
@@ -344,23 +350,62 @@ function _M:call(node_id, method, ...)
   local ok, err = fut:wait(5)
 
   if err then
-    ngx_log(ngx_DEBUG, "[rpc] ", method, " failed, err: ", err)
+    ngx_log(ngx_DEBUG, _log_prefix, method, " failed, err: ", err)
 
     return nil, err
   end
 
   if ok then
-    ngx_log(ngx_DEBUG, "[rpc] ", method, " succeeded")
+    ngx_log(ngx_DEBUG, _log_prefix, method, " succeeded")
 
     return fut.result
   end
 
-  ngx_log(ngx_DEBUG, "[rpc] ", method, " failed, err: ", fut.error.message)
+  ngx_log(ngx_DEBUG, _log_prefix, method, " failed, err: ", fut.error.message)
 
   return nil, fut.error.message
 end
 
 
+function _M:notify(node_id, method, ...)
```

Comment:
diff with call, I think we could combine them into one

```
@@ -1,4 +1,4 @@
-function _M:notify(node_id, method, ...)
+function _M:call(node_id, method, ...)
   local cap = utils.parse_method_name(method)

   local res, err = self:_find_node_and_check_capability(node_id, cap)
@@ -30,8 +30,25 @@ function _M:notify(node_id, method, ...)
   assert(res == "concentrator")

   -- try concentrator
-  local fut = future.new(node_id, self.concentrator, method, params, true)
+  local fut = future.new(node_id, self.concentrator, method, params)
   assert(fut:start())

-  return true
+  local ok, err = fut:wait(5)
+
+  if err then
+    ngx_log(ngx_DEBUG, "[rpc] ", method, " failed, err: ", err)
+
+    return nil, err
+  end
+
+  if ok then
+    ngx_log(ngx_DEBUG, "[rpc] ", method, " succeeded")
+
+    return fut.result
+  end
+
+  ngx_log(ngx_DEBUG, "[rpc] ", method, " failed, err: ", fut.error.message)
+
+  return nil, fut.error.message
 end
```

Reply:
We could do it later.

Reply:
refactored.